homeassistant/components/mqtt/strings.json

```
@@ -27,7 +27,10 @@
           "tls_insecure": "Ignore broker certificate validation",
           "protocol": "MQTT protocol",
           "set_ca_cert": "Broker certificate validation",
-          "set_client_cert": "Use a client certificate"
+          "set_client_cert": "Use a client certificate",
+          "transport": "MQTT Transport",
+          "ws_headers": "WebSockets headers in JSON format",
+          "ws_path": "WebSockets path"
```

[WebSocket](https://en.wikipedia.org/wiki/WebSocket) tends to be in singular (even if apparently the client insists on plural).
```suggestion
          "ws_headers": "WebSocket headers in JSON format",
          "ws_path": "WebSocket path"
```